doc/nrf/releases/release-notes-2.1.0.rst

```diff
@@ -0,0 +1,1087 @@
+  * Information about :ref:`gs_debugging` on the :ref:`gs_testing` page.
+    Also added links to this section in different areas of documentation.
+  * An option to configure IEEE 802.15.4 ACK frame timeout at build time using :kconfig:option:`CONFIG_NRF_802154_ACK_TIMEOUT_CUSTOM_US`.
+
+* Updated :ref:`app_memory` page with sections about Gazell and NFC.
+
+RF Front-End Modules
+--------------------
+
+* Added:
+
+  * The :kconfig:option:`CONFIG_MPSL_FEM_ONLY` Kconfig option that allows the :ref:`nrfxlib:mpsl_fem` API to be used without other MPSL features.
+    The :ref:`MPSL library <nrfxlib:mpsl>` is linked into the build without initialization.
+    You cannot use other MPSL features when this option is enabled.
+  * The possibility to add custom models which split the requested TX power between the power on SoC output and the FEM gain in a way desired by the user.
+
+* Fixed a build error that occurred when building an application for nRF53 SoCs with Simple GPIO Front-End Module support enabled.
+
+Wi-Fi
+-----
+
+* Added support for the nRF7002 DK that supports Wi-Fi SoC.
+  For more information, see the :ref:`ug_nrf70` guide.
+
+See `Wi-Fi samples`_ for details about how to use Wi-Fi in your application.
+
+Protocols
+=========
+
+This section provides detailed lists of changes by :ref:`protocol <protocols>`.
+See `Samples`_ for lists of changes for the protocol-related samples.
+
+Bluetooth LE
+------------
+
+* Added:
+
+  * Support for changing the radio transmitter's default power level using the :c:func:`sdc_default_tx_power_set` function.
+  * Support for changing the peripheral latency mode using the :c:func:`sdc_hci_cmd_vs_peripheral_latency_mode_set` function.
+  * Support for changing the default TX power using Kconfig options that start with ``CONFIG_BT_CTLR_TX_PWR_*``.
+
+* Updated LTE dual-chip Coex support on the nRF52 Series.
+  It is now ready for production.
+
+For details, see the :ref:`SoftDevice Controller changelog <nrfxlib:softdevice_controller_changelog>`.
+
+Bluetooth mesh
+--------------
+
+* Added support for using :ref:`emds_readme`.
+  For details, see `Bluetooth mesh samples`_ and `Bluetooth libraries and services`_.
+* Updated API in :ref:`bt_mesh_sensor_srv_readme`: Column get callback now gets called with a column index instead of a pointer to column.
+  This was introduced to support a series for sensors with one or two channels.
+
+Also see `Bluetooth mesh samples`_ for the list of changes.
+
+Enhanced ShockBurst (ESB)
+-------------------------
+
+* Fixed the :c:func:`update_radio_crc` function in order to correctly configure the CRC's registers (8 bits, 16 bits, or none).
+
+Matter
+------
+
+* Added support for Matter over Wi-Fi to several samples.
+* Updated :ref:`ug_matter` with new pages about Matter SDK as well as information about Matter over Wi-Fi.
+* Removed the low-power configuration build type from all Matter samples.
```

Comment:
Can we please rephrase it, as now it suggest we do not have low-power configuration.

In fact we have removed low-power configuration overlay's for few samples (Lock, Light Switch and Window Covering) and instead of requiring user to provide such overlay, we have enabled low-power communication modes by default.